examples/tutorials/ItWaspsOnMyMachine/README.md

```diff
@@ -0,0 +1,13 @@
+# It Wasps On My Machine - an app to retrieve and store excuses for all your dev needs
+
+![Sponge Bob picks an excuse for today's stand-up](../../../web/static/img/sponge-bob-excuses.jpg)
+
+It Wasps On My Machine is a simple full stack web app, that pulls dev excuses from [
+devexcuses-api Github project](https://github.com/michelegera/devexcuses-api) and enables you to save the ones you liked (and your boss don't).
+
+## Installation
+
+Clone the repo, open the terminal in the containing folder, and run:
+
+    npm install axios
```

Comment:
Why do you need this step?
Normally we would want you do specify deps via .wasp file.
Although I think Wasp might be even coming already with axios.

Reply:
Correct, you should be able to add it like so, @maksim36ua: https://github.com/wasp-lang/wasp/blob/main/examples/waspleau/main.wasp#L11

We do include `axios` @Martinsos, but only for the web app, not the server.

Reply:
@shayneczyzewski what do you think, maybe in that case we can add `axios` for server as well?